packages/transformers/js/src/JSTransformer.js

```diff
@@ -73,6 +119,8 @@ export default (new Transformer({
       return [asset];
     }
 
+    let {inlineEnvironment = true, inlineFS = true} = config || {};
```

Comment:
Did we want to change the defaults here? I thought we had decided to disable environment inlining (except `NODE_ENV`), and `fs` by default.